timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -577,14 +607,23 @@ def _stack_sample_forecasts(self, forecasts: List[SampleForecast], item_ids: pd.
     def _stack_distribution_forecasts(self, forecasts: List[Forecast], item_ids: pd.Index) -> pd.DataFrame:
```

Comment:
Would it be possible to remove the need for the many asserts by:
- changing the type hint here to `forecasts: List["DistributionForecast"]` 
- adding a "masked" import of `DistributionForecast` on top of the file with
```python
if typing.TYPE_CHECKING:
    from gluonts.torch.model.forecast import DistributionForecast
```

Maybe this would also the need to add type hints for `item_id_to_forecast` and `dist_forecasts` below?

Reply:
cool, thanks!